src/Supportive/Time/Period.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Qossmic\Deptrac\Supportive\Time;
+
+use function hrtime;
+
+final class Period
+{
+    private function __construct(
+        private readonly float|int $startedAt,
```

Comment:
I would be fine with those being public as this is basically just a ValueObject

Reply:
Done! I made the attributes public in `StartedPeriod` too.